job-service-db/src/main/resources/procedures/reportProgress.sql

```diff
@@ -17,9 +17,51 @@
 /*
  *  Name: report_progress
  *
- *  REMOVED
+ *  Description:
+ *  Reports a task progress to the database
  */
 DROP FUNCTION IF EXISTS report_progress(
     in_task_id VARCHAR(58),
     in_status job_status
 );
+CREATE OR REPLACE PROCEDURE report_progress(
+    in_partition_id VARCHAR(40),
+    in_job_id VARCHAR(58),
+    in_task_id VARCHAR(58),
+    in_task_completion DOUBLE PRECISION
+)
+    LANGUAGE plpgsql
```

Comment:
Explicitly declare that the function doesn't return anything.

Reply:
It was a procedure, which returns void by default, I changed back to function returning void explicitly